sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1385,6 +1453,14 @@ def task_worker(self):
                              self.CMIS_STATE_FAILED,
                              self.CMIS_STATE_READY,
                              self.CMIS_STATE_REMOVED]:
+                    if self.port_reinit_request_tbl[lport] is True:
+                        self.log_info("{}: Setting port_reinit_request_tbl to True".format(lport))
```

Comment:
typo？ True -> False?

Reply:
Thanks! Will correct it in the new diff.